x-pack/plugins/snapshot_restore/public/app/sections/home/repository_list/repository_details/repository_details.tsx

```diff
@@ -77,7 +77,7 @@ const RepositoryDetailsUi: React.FunctionComponent<Props> = ({
       <SectionLoading>
         <FormattedMessage
           id="xpack.snapshotRestore.repositoryDetails.loadingRepository"
-          defaultMessage="Loading repository..."
+          defaultMessage="Loading repository…"
```

Comment:
is this an intentional change?

Reply:
Thanks for checking! Yes, it is. This is better for localization purposes. I double-checked with Gail and she also approved.